src/EnergyPlus/DXCoils.cc

```diff
@@ -12730,8 +12730,9 @@ void CalcMultiSpeedDXCoilCooling(EnergyPlusData &state,
             //  Eventually inlet air conditions will be used in DX Coil, these lines are commented out and marked with this comment line
             //  AirVolumeFlowRate = AirMassFlow/PsyRhoAirFnPbTdbW(InletAirPressure,InletAirDryBulbTemp, InletAirHumRat)
             VolFlowperRatedTotCap = AirVolumeFlowRate / thisDXCoil.MSRatedTotCap(SpeedNumLS);
-            if ((VolFlowperRatedTotCap < state.dataHVACGlobal->MinOperVolFlowPerRatedTotCap(DXCT)) ||
-                (VolFlowperRatedTotCap > state.dataHVACGlobal->MaxCoolVolFlowPerRatedTotCap(DXCT))) {
+            if (((VolFlowperRatedTotCap < state.dataHVACGlobal->MinOperVolFlowPerRatedTotCap(DXCT)) ||
+                 (VolFlowperRatedTotCap > state.dataHVACGlobal->MaxCoolVolFlowPerRatedTotCap(DXCT))) &&
+                state.dataHVACGlobal->MSUSEconoSpeedNum == 0) {
```

Comment:
Ok, I see this comment. If the economizer is running then the coil air flow is on the high end of cfm/ton. As I said before this is not an issue in the real world. It may be time to remove those cfm/ton warnings and replace them with a minimum supply air temperature input field for the cooling coil models. That would be far more useful. I recently saw a -something C (-20C?) off a cooling coil in dehumidification mode. Yup, that would certainly dehumidify a zone when freezing air borne moisture onto the cooling coil.